### DISCUSSION TITLE
How should we handle CORE committee deposit information when importing to Invenio?

### DISCUSSION THREAD

**Discussion by monotasker** [MESH-Research/knowledge-commons-repository#76] [General]
CORE has two fields ("committee_deposit" and "committee_id") that seem to have been used at one point for deposits owned by a group. There are about 200 records that use these fields. For the moment I'm going to store the data in a legacy field, but what should we do with it long term? Should we be creating communities for these deposits?

The committee was never treated as the deposit submitter. The "submitter" field still holds the user id of the individual who made the upload. 

If we're going to create communities for these "committees" I'll need to get a bit more information about them. I'll need to at least have things like the title/label and members list. There are few enough that it might be easiest to do that as an export rather than by api?

**Comment by rekmla** [comment]
Note that committee membership is dynamic. Usually some percentage (10%-30%) of a committee's membership will turnover yearly.

**Comment by bjr70** [comment]
I don't know enough about this to have an opinion. Is this something we need Kathleen to weigh in on? @rekmla what was the initial reason this was recorded? Do you know? 

Do we need communities for committees? My guess is that we probably do, but I'd like to hear thoughts on that too.
